src/adafruit_blinka/microcontroller/ft232h/spi.py

```diff
@@ -67,7 +67,8 @@ def write(self, buf, start=0, end=None):
     def readinto(self, buf, start=0, end=None, write_value=0):
         """Read data from SPI and into the buffer"""
         end = end if end else len(buf)
-        result = self._port.read(end - start)
+        buffer_out = [write_value] * (end - start)
+        result = self._port.exchange(buffer_out, end - start, duplex=True)
```

Comment:
Worth wrapping this in logic?
```python
if write_value:
    result = self._port.exchange([write_value] * (end - start), end - start, duplex=True)
else:
    result = self._port.read(end - start)
```

Reply:
best to not assume anything & force the writing of the value you want. who knows if the underlying library may change